src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs

```diff
@@ -393,8 +408,15 @@ public async Task<bool> RestoreAsync(SolutionRestoreRequest request, Cancellatio
                     using (var restoreOperation = new BackgroundRestoreOperation())
                     {
                         await PromoteTaskToActiveAsync(restoreOperation, token);
-
-                        var result = await ProcessRestoreRequestAsync(restoreOperation, request, token);
+                        var restoreTrackingData = GetRestoreTrackingData(
+                            restoreReason: ImplicitRestoreReason.None,
+                            requestCount: 1,
+                            isBulkRestoreCoordinationEnabled: true,
```

Comment:
Do we need to check if experimentation is enabled here before assign `true` to `isBulkRestoreCoordinationEnabled`?

Reply:
Not super relevant for our data, but fixed for completness.